### DISCUSSION TITLE
Aide

### DISCUSSION THREAD

**Discussion by Ramadane10** [community/community#140506] [Discussions]
### Select Topic Area

Question

### Body

Comment faire pour recevoir le sms de double authentification si ton pays ne figure pas sur la liste

**Comment by LiteBrite82** [comment]
Hi @Ramadane10 , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
